awot/dropsondes/dropsondes.py

```diff
@@ -8,651 +8,698 @@
+    ax1.yaxis.set_major_formatter(ScalarFormatter())
+    ax1.set_yticks(np.linspace(100, 1000, 10))
+    ax1.set_ylim(1050, 100)
+    ax1.set_ylabel('Pressure mb')
+    ax1.set_xlabel('Temperature C')
+    ax1.xaxis.set_major_locator(MultipleLocator(10))
+    ax1.set_title('Skew T log P')
 
-        self.ax1.yaxis.set_major_formatter(ScalarFormatter())
-        self.ax1.set_yticks(np.linspace(100, 1000, 10))
-        self.ax1.set_ylim(1050, 100)
-        self.ax1.set_ylabel('Pressure mb')
-        self.ax1.set_xlabel('Temperature C')
-        self.ax1.xaxis.set_major_locator(MultipleLocator(10))
-        self.ax1.set_title('Skew T log P')
+    y_min = kwargs.get('y_min')
+    y_max = kwargs.get('y_max')
 
-        self.y_min = kwargs.get('y_min')
-        self.y_max = kwargs.get('y_max')
+    x_min = kwargs.get('x_min')
+    x_max = kwargs.get('x_max')
 
-        self.x_min = kwargs.get('x_min')
-        self.x_max = kwargs.get('x_max')
+    # data label, title and font sizes value assignments
+    # default values above.
 
-        # data label, title and font sizes value assignments
-        # default values above.
+    y_label = kwargs.get('y_label')
+    x_label = kwargs.get('x_label')
+    title = kwargs.get('title')
+    font_size = kwargs.get('font_size')
 
-        y_label = kwargs.get('y_label')
-        x_label = kwargs.get('x_label')
-        title = kwargs.get('title')
-        font_size = kwargs.get('font_size')
+    for item in ([ax1.xaxis.label, ax1.yaxis.label] +
+                 ax1.get_xticklabels()+ax1.get_yticklabels()):
+        item.set_fontsize(font_size)
 
-        for item in ([self.ax1.xaxis.label, self.ax1.yaxis.label] +
-                     self.ax1.get_xticklabels()+self.ax1.get_yticklabels()):
-            item.set_fontsize(font_size)
+    # set the labels and titles defaults to 'none'
 
-        # set the labels and titles defaults to 'none'
+    ax1.set_ylabel(y_label)
+    ax1.set_xlabel(x_label)
+    ax1.set_title(title)
 
-        self.ax1.set_ylabel(y_label)
-        self.ax1.set_xlabel(x_label)
-        self.ax1.set_title(title)
+    ax1.set_ylim(y_max, y_min)
+    ax1.set_xlim(x_min, x_max)
 
-        self.ax1.set_ylim(self.y_max, self.y_min)
-        self.ax1.set_xlim(self.x_min, self.x_max)
+def plot_hodograph(data):
 
-    def plot_hodograph(self, data):
+    '''
+    method to plot wind data on a hodograph
 
-        '''
-        method to plot wind data on a hodograph
+    Inputs:
 
-        Inputs:
+    dictionary containing wind data.
 
-        dictionary containing wind data.
+    file path
+    # ============
 
-        file path
-        # ============
+    output:
 
-        output:
+    # Image
 
-        # Image
+    '''
 
-        '''
+    ax2 = fig.add_axes([.05, 0.6, 0.25, 0.3])
+
+    # create axis and invert masks and assign values
+    # for U, V, and h coordinates
+
+    U = data['U Component']
+    V = data['V Component']
+    H = data['Height']
+    
```

Comment:
If we standardized the names with the other portions of AWOT (e.g. I used Uwind I think), some of these routines would become more general.